.github/workflows/auto-merge.yaml

```diff
@@ -0,0 +1,24 @@
+name: AutoMerge 💥
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, reopened, synchronize]
```

Comment:
We don't want the automerge workflow to be run on the PR closed event (unlike test)